### PR TITLE
gtk3.20: Numix sidebar, Treeview progressbar and Notebook minimal fix.

### PR DESCRIPTION
**Pull request by megax**
Nemo:
image

Progressbar:
image

### CODE REVIEW

**Comment by megax** [comment]
@lots0logs Nemo, progressbar fix. :)

**Comment by lots0logs** [comment]
@megax What exactly does this fix in Nemo? I can't tell from your screenshot :grimacing: I won't have time to review these until tonight...

**Comment by megax** [comment]
NemoPlacesTreeView-disk-full settings. Old style:
image

The disk sider part of Nemo wasn't responding to changes so I replaced it so it can work and changed it's style to look better.

**Comment by megax** [comment]
I repaired in the Notebook, when selecting the tab, it jumped 1px. It's temporary, needs some change. The close button's size is fixed. There was a 1px padding on(vagy under nem tudom melyik lenne jó) Nemo desktop and gnome-terminal and on some other application, I fixed that too. :)

**Comment by megax** [comment]
Fixed switch padding:
Old:
image

New:
image

**Comment by lots0logs** [comment]
@megax I designed the new switch myself. Other than a few minor adjustments that I still need to do to it, I don't want to change it.

**Comment by megax** [comment]
Ok, revert commit :)

**Comment by megax** [comment]
Done patch :)

**Comment by megax** [comment]
Minimal fix... :)

File: gtk-3.20/scss/widgets/_notebook.scss [hunk 1]
```diff
@@ -7,22 +7,17 @@
 
 @include exports("notebook") {
     notebook {
-        padding: 0;
-        border-width: 1px 0 0;
-        border-style: solid;
-        border-color: border_normal($base_color);
-        border-radius: 0;
         background-color: $base_color;
```
Comment (megax) [inline gtk-3.20/scss/widgets/_notebook.scss:10]: Not optimal background color fix Nemo... Other application corrupted... (Rhythmbox)

Comment (lots0logs) [inline gtk-3.20/scss/widgets/_notebook.scss:10]: @megax So we need to revert this change right? I think it will be easier (for the both of us) going forward if you break changes up into separate pull requests. Something else that would be really helpful (though you don't have to do it if you don't want) is to include before and after screenshots of the changes for each pull request :grimacing:

Comment (megax) [inline gtk-3.20/scss/widgets/_notebook.scss:10]: @lots0logs Yes, background-color change revert (https://github.com/shimmerproject/Numix/commit/2eb312308438f6b51792854466406dd81bafe81a). Nemo fix later... :)
New pull requests separate ok :) and before, after screenshots :P

**Comment by megax** [comment]
@lots0logs I close the pull request?

**Comment by lots0logs** [comment]
Yes I got all the changes from this pull request. I will revert the background change in a few minutes. Thanks! :smile:
